modules/virt-creating-vm-custom-template.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-vm-custom-template_{context}"]
+= Creating a virtual machine from a customized template
+
+Some templates require additional parameters, for example, a PVC with a boot source. You can customize select parameters of a template to create a virtual machine (VM).
+
+.Procedure
+
+. Click *Virtualization* > *Catalog* in the side menu.
+
++
+Optional: Filter the templates by operating system or workload profile.
+
+. Click the template and then click *Customize VirtualMachine* to modify the template, for example, by specifying a boot source, the disk size, or a DataSource to clone.
+
+. Click *Review and create VirtualMachine* and edit the template parameters.
+
+. Click *Create VirtualMachine*.
++
+The VM *Details* page is displayed with the provisioning status.
+
+. Click *Events* to verify the progress of the VM provisioning.
```

Comment:
The Events page doesn't show a lot if it provisioned successfully, so I might rephrase this a bit. Also, these last two steps should probably be in their own verification section. Suggestion:

```
.Verification

. Click *Events* to view a stream of events as the VM is provisioned.

. Click *Console* to verify that the VM booted successfully.
```